lib/rubygems/ext/rake_builder.rb

```diff
@@ -9,7 +9,7 @@ class Gem::Ext::RakeBuilder < Gem::Ext::Builder
 
   def self.build(extension, directory, dest_path, results, args=[], lib_dir=nil)
     if File.basename(extension) =~ /mkrf_conf/i then
-      cmd = "#{Gem.ruby} #{File.basename extension}"
+      cmd = "#{Gem.ruby} #{File.basename extension}".dup
       cmd << " #{args.join " "}" unless args.empty?
```

Comment:
I don't have authority to accept PR's, sorry. Ask @segiddins or other RG core team members. I'm sure they won't mind since it's a low risk fix.